app/src/main/res/layout/item_member.xml

```diff
@@ -18,14 +18,24 @@
         app:layout_constraintLeft_toLeftOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
+    <ImageView
+        android:id="@+id/image_member_status"
+        android:layout_width="12dp"
+        android:layout_height="0dp"
+        android:layout_marginStart="16dp"
+        app:layout_constraintBottom_toBottomOf="@+id/layout_avatar"
+        app:layout_constraintStart_toEndOf="@+id/layout_avatar"
+        app:layout_constraintTop_toTopOf="@+id/layout_avatar"
+        tools:src="@drawable/ic_status_invisible_12dp" />
+
     <TextView
```

Comment:
You can use `drawableLeft` instead of declaring this `ImageView` here. 
([doc](https://developer.android.com/reference/android/widget/TextView.html#attr_android:drawableLeft))

Reply:
Ok, I've changed the code to use `drawableStart` and its related method `setCompoundDrawablesRelativeWithIntrinsicBounds`.